conda/env/env.py

```diff
@@ -265,15 +264,6 @@ def save(self):
             self.to_yaml(stream=fp)
 
 
-def get_filename(filename):
```

Comment:
not sure if this needs to be deprecated instead of removed like this.

Reply:
We need to deprecate it.